services/web/server/src/simcore_service_webserver/studies_dispatcher/_models.py

```diff
@@ -18,7 +19,9 @@ class ServiceInfo(BaseModel):
 
     label: str = Field(..., description="Display name")
 
-    thumbnail: HttpUrl = Field(default="https://via.placeholder.com/170x120.png")
+    thumbnail: HttpUrl = Field(
+        default=cast(HttpUrl, "https://via.placeholder.com/170x120.png")
```

Comment:
for this instead of casting I think using the HttpUrl constructor makes more sense.
HttpUrl is a str with additional constraints. Using cast here means that in the (I agree remote) case where the default value is wrongly set, you will not detect it.

Reply:
Thanks didn't catch this one, i will replace with `parse_obj_as`